openstudiocore/src/model/Model.hpp

```diff
@@ -87,6 +88,9 @@ class MODEL_API Model : public openstudio::Workspace {
   /** @name Getters */
   //@{
 
+  /// Get the WorkflowJSON
```

Comment:
There are so many seemingly unrelated things in this PR. Is that intentional? A git error? or is all related somehow? Looks to me like you might be bringing some other stuff from develop since the 2.0 branch was last caught up.